paddle/fluid/framework/operator.cc

```diff
@@ -1007,21 +1007,23 @@ static void CheckTensorNANOrInf(const std::string& op_type,
                               op_type, name));
 }
 
-bool OperatorWithKernel::SupportsMKLDNN() const {
+bool OperatorWithKernel::SupportsMKLDNN(const framework::ExecutionContext& ctx,
+                                        proto::VarType::Type data_type) const {
   auto& op_kernels = OperatorWithKernel::AllOpKernels().at(type_);
   return std::any_of(op_kernels.begin(), op_kernels.end(),
-                     [](OpKernelMap::const_reference kern_pair) {
+                     [ctx, data_type](OpKernelMap::const_reference kern_pair) {
```

Comment:
The catched variable `ctx` is not used in lambda body.

Reply:
Done